src/ExpressServer.ts

```diff
@@ -0,0 +1,44 @@
+import express from "express";
+import { Environment } from "./utils/constants";
+import { SetHueTokens } from "./utils/helpers";
+
+export default class ExpressServer {
+    private _server;
+
+    constructor() {
+        this._server = express();
+
+        this._server.get("/", (request, response) => {
+            response.send("Discord bot active.");
+        });
+        
+        // hue auth flow configuration
+        this._server.get("/seabot_hue", async (request, response) => {
```

Comment:
Note for future PRs / dev, we can likely also stub out these individual response functions into separate files - which will likely become much more useful if we end up with any sort of web interface with routing